config/Dockerfiles/tests.d/aws/03_aws_ec2_vpc_subnet

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
 # Verify the aws ec2 vpc subnet provisioning
-# distros.exclude: centos7 fedora28 fedora30
+# distros.exclude: centos7 fedora29 fedora30
```

Comment:
We should still be running this on at least one distro

Reply:
@14rcole subnet is tested as part of route-table test 
routetable provisions subnet we need not test it again